custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -148,10 +148,10 @@ func main() {
 	flags.AddGoFlagSet(flag.CommandLine) // make sure we get the klog flags
 
 	serverOptions := stackdriverAdapterServerOptions{
```

Comment:
Why do we change the default values?

Reply:
I was proposing to support prometheus_target by default without adding new flags.

Reply:
as discussed, I will revert this change